packages/flutter_tools/lib/src/commands/analyze_continuously.dart

```diff
@@ -53,7 +54,10 @@ class AnalyzeContinuously extends AnalyzeBase {
     await server.start();
     final int exitCode = await server.onExit;
 
-    printStatus('Analysis server exited with code $exitCode.');
+    String message = 'Analysis server exited with code $exitCode.';
+    if (exitCode != 0)
+      throw new ToolExit(message, exitCode: exitCode);
```

Comment:
Good point. I want a method name that makes it obvious that I'm throwing an exception so `throwToolExit`.